docs/sink/flink-connector-obkv-hbase_cn.md

```diff
@@ -139,6 +186,28 @@ CREATE TABLE t_sink
   'sys.password'='');
 ```
 
+##### odp模式
+
+```sql
+CREATE TABLE t_sink
+(
+  rowkey STRING,
+  family1 ROW <column1 STRING,
+  column2 STRING >,
+  PRIMARY KEY (rowkey) NOT ENFORCED
+) with (
+  'connector'='obkv-hbase',
+  'odp-mode'='true',
+  'odp-ip'='127.0.0.1',
+  'odp-port'='12881',
+  'schema-name'='test',
+  'table-name'='htable1',
+  'username'='root@test',
+  'password'='',
+  'sys.username'='root',
```

Comment:
Remove sys username and password.

Reply:
Instructions have been added，sys username and sys password you can leave it blank

Reply:
It's better to delete it if it's not in use.